architecture/additional_concepts/authorization.adoc

```diff
@@ -605,21 +605,27 @@ These examples are in the context of a strategy using the preallocated values.
 *A FSGroup SCC Strategy of MustRunAs*
 
 If the pod defines a `*fsGroup*` ID, then that ID must equal the default
-`*FSGroup*` ID. Otherwise, the pod is not validated by that SCC and the next SCC
-is evaluated. If the `*FSGroup*` strategy is *RunAsAny* and the pod omits a
-`*fsGroup*` ID, then the pod matches the SCC based on `*FSGroup*` (though other
-strategies may not validate and thus cause the pod to fail).
+`*fsGroup*` ID. Otherwise, the pod is not validated by that SCC and the next SCC
+is evaluated.
+
+If the `*SecurityContextConstraints.fsGroup*` field has value *RunAsAny*
+and the pod specification omits the `*Pod.spec.securityContext.fsGroup*`,
+then this field is considered valid. Note that we talking about a single field
```

Comment:
Do we need this phase: "we talking about a single field of a pod"?
"Note, it is possible that during validation, other SCC settings will reject other pod fields and thus cause the pod to fail."
If so, change below also.